apps/staking/hooks/parseContracts.ts

```diff
@@ -153,7 +149,7 @@ export function parseContracts({
       const lastFinalized = events.filter((event) => event.name === 'Finalized')[0];
       if (!lastFinalized) {
         logger.warn(`Contract is finalized, but no Finalized event, showing: ${pubkey_bls}`);
-      } else if (lastFinalized.block < nodeMinLifespanArbBlocks) {
+      } else if (lastFinalized.block > nodeMinLifespanArbBlocks) {
         logger.debug(
           `Contract was finalized at block ${lastFinalized.block}, this is less than the lifespan limit (${nodeMinLifespanArbBlocks}), showing: ${pubkey_bls}`
```

Comment:
I'm confused. Is this still "less than the lifespan limit" ?

Reply:
updated comment